extractor/src/main/java/org/schabi/newpipe/extractor/utils/LocaleCompat.java

```diff
@@ -0,0 +1,36 @@
+package org.schabi.newpipe.extractor.utils;
+
+import java.util.Locale;
+
+public class LocaleCompat {
```

Comment:
Could you add some documentation that this class is made to keep compatibility with Android old APIs? Also maybe precise that if the missing method is made later available with desugarging, this way should be preferred.